dev-lang/lfe/lfe-1.2.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Could you port it to EAPI 6, please? We always want to use the latest stable EAPI for the new ebuilds.